src/octopeer-github/main/MainController.ts

```diff
@@ -113,15 +111,20 @@ class MainController implements OptionsObserver {
 
     /**
      * When a tab sends a message, log it to the Database.
+     * If a tab sends a message while having a URL that is invalid, send "unhookFromDom" (defensive programming).
      */
     private listenToDatabaseMessages() {
         chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
             if (!sender.tab) {
                 return; // Only continue if message is sent from a content script
             }
-            // IP for testing locally: 10.0.22.6
-            this.database = new RESTApiDatabaseAdapter("http://146.185.128.124", sender.tab.url, this.user);
-            this.postToDatabase(this.readMessage(message));
+            if (URLHandler.isPullRequestUrl(sender.tab.url)) {
+                this.postToDatabase(this.readMessage(message), sender.tab.url);
+            } else {
+                Logger.warn(`Squelching ${sender.tab} after it sent:`);
```

Comment:
Google says:
> _informal_
> forcefully silence or suppress.
> "property developers tried to squelch public protest"
When a message is sent from something that is not a pull request, it should send a message that lets it unhook from DOM.